lib/include/openamp/virtio.h

```diff
@@ -275,6 +275,12 @@ struct virtio_dispatch {
 
 	/** Notify the other side that a virtio vring as been updated. */
 	void (*notify)(struct virtqueue *vq);
+
+	/**
+	 * Customize the wait, when no Tx buffer is available, optional
```

Comment:
would prefer `Customize the wait, when no Tx buffer is available (optional)`

Reply:
Done